solutions/instances/variables.tf

```diff
@@ -30,16 +30,10 @@ variable "existing_monitoring_crn" {
 # KMS variables
 ########################################################################################################################
 
-variable "kms_region" {
-  type        = string
-  default     = "us-south"
-  description = "The region in which KMS instance exists."
-}
-
-variable "existing_kms_guid" {
+variable "existing_kms_instance_crn" {
   type        = string
   default     = null
-  description = "The GUID of of the KMS instance used for the SCC COS bucket root Key. Only required if not supplying an existing KMS root key and if 'skip_cos_kms_auth_policy' is true."
+  description = "The CRN of the existed Hyper Protect Crypto Services or Key Protect instance. Only required if not supplying an existing KMS root key and if 'skip_cos_kms_auth_policy' is true."
 }
 
 variable "existing_scc_cos_kms_key_crn" {
```

Comment:
can you update the description of this to say that if no value is passed, a value must be passed for either `existing_kms_instance_crn` if you want to create a new key ring and key, or `existing_scc_cos_bucket_name` if you want to use an existing bucket